d2core/d2ui/widget.go

```diff
@@ -11,6 +11,8 @@ const (
 	RenderPrioritySkilltree
 	// RenderPrioritySkilltreeIcon is the priority for the skilltree icons
 	RenderPrioritySkilltreeIcon
+	// RenderPriorityMinipanel is the priority for the minipanel icons
+	RenderPriorityMinipanel
 	// RenderPriorityHeroStatsPanel is the priority for the hero_stats_panel
 	RenderPriorityHeroStatsPanel
 	// RenderPriorityForeground is the last element drawn
```

Comment:
This is fine how it is, but for future reference; to avoid the pedantic comments, you can put a comment at the top of this const block to satisfy the lint god (it must start with a capital letter).

Reply:
Oh, I didn't know. This is much better. Thanks.